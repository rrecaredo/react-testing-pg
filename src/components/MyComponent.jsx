import React from "react";

export default () => <React.Fragment>Hello</React.Fragment>;
