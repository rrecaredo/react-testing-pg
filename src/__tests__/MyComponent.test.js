import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import MyComponent from "../components/MyComponent";

afterEach(cleanup);

describe("something", () => {
  test("renders", () => {
    const { asFragment } = render(<MyComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
