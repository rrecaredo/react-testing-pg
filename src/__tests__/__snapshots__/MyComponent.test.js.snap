// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`something renders 1`] = `
<DocumentFragment>
  Hello
</DocumentFragment>
`;
